src/game-data/effects/cards/PR-132.ts

```
@@ -0,0 +1,25 @@
+
+export const effects: CardEffects = {
+  onDriveSelf: async (stack: StackWithCard<Unit>): Promise<void> => {
+    const targets = stack.processing.owner.opponent.field.filter(unit =>
+      unit.hasKeyword('秩序の盾')
+    );
+
+    await EffectHelper.combine(stack, [
+      {
+        title: '無秩序の世界',
+        description: '【秩序の盾】を破壊',
+        effect: () => targets.forEach(target => Effect.break(stack, stack.processing, target)),
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)